src/components/searchbar/Searchbar.jsx

```diff
@@ -1,18 +1,87 @@
 import "./Searchbar.scss";
 import { useEffect, useState } from "react";
+import axios from "axios";
 
-const Searchbar = ({ cat }) => {
+const Searchbar = ({ media, genreId }) => {
   const [search, setSearch] = useState("");
   const [display, setDisplay] = useState("");
 
+  const genres = {
+    28: "Action",
+    10759: "Action & Adventure",
+    12: "Adventure",
+    16: "Animation",
+    35: "Comedy",
+    80: "Crime",
+    99: "Documentary",
+    18: "Drama",
+    10751: "Family",
+    14: "Fantasy",
+    36: "History",
+    27: "Horror",
+    10762: "Kids",
+    10402: "Music",
+    9648: "Mystery",
+    10763: "News",
+    10764: "Reality",
+    10749: "Romance",
+    878: "Science Fiction",
+    10765: "Sci-Fi & Fantasy",
+    10766: "Soap",
+    10767: "Talk",
+    10770: "TV Movie",
+    53: "Thriller",
+    10752: "War",
+    10768: "War & Politics",
+    37: "Western",
+  };
+
+  const apiValues = {};
+  if (genreId) {
+    for (const key in genres) {
+      if (key.includes(genreId)) {
+        apiValues.genre = genres[key].toLowerCase();
+      }
+    }
+    switch (media) {
+      case "movie":
+        apiValues.url = `https://api.themoviedb.org/3/discover/movie?api_key=c27f8ca565d1e0e0b28f3721fb54e05e&with_genres=${genreId}`;
```

Comment:
Les clés d'API ne doivent surtout pas être en dur comme ça
-> Check les [variables d'environnement](https://vitejs.dev/guide/env-and-mode.html#env-files)

Reply:
Ok pour ça